pubspec.yaml

```diff
@@ -31,7 +31,6 @@ dependencies:
   flutter:
     sdk: flutter
 
-
   # The following adds the Cupertino Icons font to your application.
   # Use with the CupertinoIcons class for iOS style icons.
   cupertino_icons: ^1.0.2
```

Comment:
You are adding cupertino_icons, you are going to run it on iPhone or do you only want to use cupertino icons?

Reply:
Thanks for the heads up! 
It was added by default by following the Onboarding guides on project creation.
Now i know that it is not necessary, and will remove it from pubspec, and stick with Material, because we will only work with Android **_as far as i know_**.